planning/behavior_velocity_planner/src/node.cpp

```diff
@@ -309,7 +309,7 @@ void BehaviorVelocityPlannerNode::onVehicleVelocity(
   // Add velocity to buffer
   planner_data_.velocity_buffer.push_front(*current_velocity);
```

Comment:
I think velocity_buffer size is always more than one at this line. is there any cases this would happen?